versions/0.1.md

```diff
@@ -0,0 +1,107 @@
+# Conventional Changelog Configuration Spec (v0.1)
+
+## Structure
+
+- [types](#types)
+  - [type](#type)
+- [commitUrlFormat](#commiturlformat-string)
+- [compareUrlFormat](#compareurlformat-string)
+- [issueUrlFormat](#issueurlformat-string)
+
+
+## Substitutions
+- [host](#host)
+- [owner](#owner)
+- [repository](#repository)
+- [hash](#hash)
+- [previousTag](#previoustype)
+- [currentTag](#currenttag)
+
+---
+
+### types
+
+An array of [`type`](#type) objects representing the explicitly supported commit message types, and whether they should show up in generated CHANGELOGs.
+
+```json
+"types": [
+    { "type": "feat", "section": "Features"},
+    { "type": "fix", "section": "Bug Fixes"},
+    { "type": "test", "section": "Tests"},
+    { "type": "build", "section": "Build System"},
+    { "type": "ci", "hidden": true}
+]
+```
+
+#### type
+
+| name                      | type      | required  | default | description  |
+| ------------------------- | --------- | --------- | ------- | ----------------------------------------------------------------------------------------------------------------- |
+| **type**                  | `string`  | ✔️        | `N/A`   | A string used to match `<type>`s used in the [Conventional Commits](https://www.conventionalcommits.org) convention. |
+| **section**               | `string`  | ✖️        | `N/A`   | The section where the matched commit `type` will display in the CHANGELOG. |
+| **hidden**                | `boolean` | ✖️        | `N/A`   | Set to `true` to hide matched commit `type`s in the CHANGELOG.                     |
+
+### commitUrlFormat (`string`)
+
+A URL representing a specific commit at a hash.
+
+```
+{{host}}/{{owner}}/{{repository}}/commit/{{hash}}
+```
+
+See [Substitutions]() for more details on substitutions.
+
+### compareUrlFormat (`string`)
+
+A URL representing the comparison between two git shas.
+
+```
+{{host}}/{{owner}}/{{repository}}/compare/{{previousTag}}...{{currentTag}}
+```
+
+See [Substitutions]() for more details on substitutions.
+
+### issueUrlFormat (`string`)
+
+A URL representing the issueformat (allowing a different URL format to be swapped in for Gitlab, Bitbucket, etc):
+
+```
+{{host}}/{{owner}}/{{repository}}/issues/{{id}}
```

Comment:
one URL format we're missing is `userUrlFormat`, used to bring up a user-account from a `@bcoe` in the commit message.